src/mpid/ch4/netmod/ofi/ofi_win.h

```diff
@@ -1062,6 +1068,8 @@ static inline int MPIDI_NM_mpi_win_allocate_shared(MPI_Aint size,
         shm_unlink(shm_key);
 
   fn_exit:
+    if (shm_key != NULL)
```

Comment:
I agree for `free()` (just double checked in the `man` page), but not so sure about `MPL_free()`. I tried without the check but one test was failing on `MPL_free()`. 

The failed check was `rma/win_zero`.